test/test_helper.exs

```diff
@@ -1 +1,7 @@
 ExUnit.start(exclude: [:system])
+
+{:ok, files} = File.ls("./test/helper")
```

Comment:
THis can be done the same way default Phoenix projects are configured.
In mix.exs you can do
```elixir
def project do
  [...
   elixirc_paths: elixirc_paths(Mix.env),
  ...]
end

defp elixirc_paths(:test), do: ["lib/", "test/helper/"]
defp elixirc_paths(_), do: ["lib/"]
```